src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandler.java

```diff
@@ -192,8 +194,13 @@ public void onData(Stream stream, DataFrame frame, Callback callback) {
                 result.latencyEnd();
                 first = false;
             }
+
```

Comment:
Following the same implementation path as JMeter with HTTP plugin, a check-box for gzip decompression will not be needed. The plugin for itself should recognize when a request is GZIP format.

In this class we can manage to recognize if is GZIP or not without the user confirmation (checkbox).
If we add something like: 
```java
  result.setGzip(frame.getMetaData().getFields().get(HttpHeader.CONTENT_ENCODING)
            .equals(HTTPConstants.ENCODING_GZIP));
```
Somewhere [**here**](https://github.com/Blazemeter/jmeter-http2-plugin/blob/6de6898546397de7b67af3238b5f7f3aa74409ec/src/main/java/com/blazemeter/jmeter/http2/sampler/HTTP2StreamHandler.java#L173-#L175). If the link is not working, it should be added on method `onHeaders(Stream, HeadersFrame)` after the `result.setHttpFieldsResponse(frame.getMetaData().getFields());`